dbt/include/mariadb/macros/materializations/snapshot/create_columns.sql

```diff
@@ -0,0 +1,14 @@
+{#
+    Add new columns to the table if applicable
+#}
+{% macro create_columns(relation, columns) %}
+  {{ adapter.dispatch('create_columns', 'dbt')(relation, columns) }}
+{% endmacro %}
+
+{% macro default__create_columns(relation, columns) %}
```

Comment:
The `default_` prefix is used in `dbt-core` as a prefix for the default implementation of a macro. Any adapter specific macros should be prefixed with their adapter name (`mariadb_` in this case). 
```suggestion
{% macro mariadb__create_columns(relation, columns) %}
```